project/Build.scala

```diff
@@ -2370,10 +2370,11 @@ object ScaladocConfigs {
       .add(VersionsDictionaryUrl("https://scala-lang.org/api/versions.json"))
       .add(DocumentSyntheticTypes(true))
       .add(SnippetCompiler(List(
-        s"$dottyLibRoot/src/scala=compile",
+        s"${dottyLibRoot}/scala=compile",
```

Comment:
This change seems suspicious. IIUC it *removes* things from the set of things that are compiled. Was it intentional? If yes, why this change?

Reply:
Sorry, that was just a mistake. I undo the change.

Reply:
done,